README.md

```diff
@@ -450,7 +460,7 @@ a.versions.length                         # 2
 a.paper_trail.previous_version.content    # nil
 ```
 
-The `:ignore` and `:only` options can also accept `Hash` arguments.
+The `:only` option can also accept `Hash` arguments that we are considering deprecating.
```

Comment:
I think the example that follows this line is still incorrect. It shows `only: {}`, but should be `only: []`, right?

Reply:
Thank you!  Updated the example 👍